modxcms/revolution
# Enable custom number of maximum search results

### What does it do?
Controls the maximum number of search results returned for each element type when using the search function in the manager menu.

### Why is it needed?
The maximum number was hard-coded to 5. This is often not enough to be useful.

### How to test
Install this update, then edit the new system setting "search_max_results". You should now see the number of results you specified when using the search tool.

### Related issue(s)/PR(s)
https://community.modx.com/t/search-in-manager-limited/3960
